king_phisher/server/database/manager.py

```diff
@@ -281,6 +289,16 @@ def init_database(connection_url, extra_init=False):
 		if not alembic_directory:
 			raise errors.KingPhisherDatabaseError('cannot find the alembic data directory')
 
+		if not 'alembic_version' in inspector.get_table_names():
+			alembic_metadata = sqlalchemy.MetaData(engine)
+			alembic_table = sqlalchemy.Table('alembic_version', alembic_metadata,
```

Comment:
The whitespace here is off and this block should probably all be put in a new `init_alembic` function.